augly/image/functional.py

```diff
@@ -1786,3 +1786,56 @@ def vflip(
     imutils.get_metadata(metadata=metadata, function_name="vflip", **func_kwargs)
 
     return imutils.ret_and_save_image(aug_image, output_path)
+
+
+def skew(
+    image: Union[str, Image.Image],
+    output_path: Optional[str] = None,
+    level: float = 5.0,
+    max_level: float = 10.0,
+    axis: str = "x_axis",
+    metadata: Optional[List[Dict[str, Any]]] = None,
+) -> Image.Image:
+    
+    """
+    Skew image with respect to X-axis or Y-axis
+    
+    @param image: the path to an image or a variable of type PIL.Image.Image
+        to be augmented
+        
+    @param level: Level of the operation that will be in [0, `PARAMETER_MAX`]
```

Comment:
What is `PARAMETER_MAX`? You never define it.

Reply:
Parameter max is the maximum value of level user want to skew the image.

Reply:
I've made some changes regarding the Parameter max. If seems okay then I can resolve this comment too.

Reply:
What changes?